docs/core/testing/unit-testing-mstest-writing-tests-assertions.md

```diff
@@ -12,12 +12,45 @@ Use the `Assert` classes of the <xref:Microsoft.VisualStudio.TestTools.UnitTesti
 
 ## The `Assert` class
 
-In your test method, you can call any methods of the <xref:Microsoft.VisualStudio.TestTools.UnitTesting.Assert?displayProperty=fullName> class, such as <xref:Microsoft.VisualStudio.TestTools.UnitTesting.Assert.AreEqual%2A?displayProperty=nameWithType>. The <xref:Microsoft.VisualStudio.TestTools.UnitTesting.Assert> class has many methods to choose from, and many of the methods have several overloads.
+Use the <xref:Microsoft.VisualStudio.TestTools.UnitTesting.Assert> class to verify that the code under test behaves as expected. Available APIs are:
+
+- <xref:Microsoft.VisualStudio.TestTools.UnitTesting.Assert.AreEqual>
+- <xref:Microsoft.VisualStudio.TestTools.UnitTesting.Assert.AreNotEqual>
+- <xref:Microsoft.VisualStudio.TestTools.UnitTesting.Assert.AreNotSame>
+- <xref:Microsoft.VisualStudio.TestTools.UnitTesting.Assert.AreSame>
+- <xref:Microsoft.VisualStudio.TestTools.UnitTesting.Assert.Fail>
+- <xref:Microsoft.VisualStudio.TestTools.UnitTesting.Assert.Inconclusive>
+- <xref:Microsoft.VisualStudio.TestTools.UnitTesting.Assert.IsFalse>
+- <xref:Microsoft.VisualStudio.TestTools.UnitTesting.Assert.IsInstanceOfType>
+- <xref:Microsoft.VisualStudio.TestTools.UnitTesting.Assert.IsNotInstanceOfType>
+- <xref:Microsoft.VisualStudio.TestTools.UnitTesting.Assert.IsNotNull>
+- <xref:Microsoft.VisualStudio.TestTools.UnitTesting.Assert.IsNull>
+- <xref:Microsoft.VisualStudio.TestTools.UnitTesting.Assert.IsTrue>
+- <xref:Microsoft.VisualStudio.TestTools.UnitTesting.Assert.ThrowsException>
+- <xref:Microsoft.VisualStudio.TestTools.UnitTesting.Assert.ThrowsExceptionAsync>
```

Comment:
The new `Throws` are 3.8+ and not yet available I think? We will need to remember updating this.

Reply:
Yep!